src/main/java/org/openlmis/stockmanagement/domain/sourcedestination/SourceDestinationAssignment.java

```diff
@@ -38,4 +38,7 @@
   @ManyToOne()
   @JoinColumn(nullable = false)
   Node node;
+  
+  @Type(type = PG_UUID)
+  UUID geoLevelAffinity;
```

Comment:
I am not sure if the given field name is correct. If this is ID of geographic level then we should simply call it *geographicLevelId* same if this is ID of the geographic zone.

Reply:
I agree this should have the "id" suffix, but it doesn't necessarily need to be called geographicLevel or geographicZone as it's not descriptive at all